testsuite/features/step_definitions/command_steps.rb

```diff
@@ -21,13 +21,42 @@
   $server.run("ping -c1 #{node.full_hostname}")
 end
 
-Then(/^it should be possible to download the file "([^"]*)"$/) do |file|
-  $server.run("curl -k #{file} -o /dev/null")
-end
-
-Then(/^it should be possible to reach the (.*) registry$/) do |registry|
-  url = registry == 'portus' ? 'https://portus.mgr.suse.de:5000' : 'https://registry.mgr.suse.de:443'
-  $server.run("curl -k --fail #{url}")
+Then(/^it should be possible to reach the (.*)$/) do |resource|
+  case resource
+  when 'download site'
+    if $product == 'Uyuni'
+      url = 'https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Test-Packages:/Updates/rpm/repodata/repomd.xml'
```

Comment:
Personally, I would prefer to have those strings in /testsuite/features/support/constants.rb

Reply:
so it becomes more impractical because you have to change file ? :-)

Reply:
Some day I will show you how nice and practical can be to use a real development IDE :trollface: